xpsi/Sample.py

```diff
@@ -65,8 +69,6 @@ def ensemble(likelihood, prior, MPI = True, **kwargs):
                 pool.wait()
                 _sys.exit(0)
 
-            from xpsi.EnsembleSampler import EnsembleSampler
```

Comment:
I guess removal of emcee imports are breaking the GitHub Action tests. Did you remove these by purpose?

Reply:
Well, at least the error messages are complaining about emcee:
https://github.com/xpsi-group/xpsi/actions/runs/9643508619/job/26593654031

Reply:
Okay, I found the problem is actually the re-location of the emcee import that I mentioned below.